src/main/java/com/dnsimple/endpoints/Certificates.java

```diff
@@ -131,28 +131,12 @@ public SimpleResponse<Certificate> issueLetsencryptCertificate(Number account, S
      * @param account       The account ID
      * @param domain        The domain name or ID
      * @param certificateId The certificate ID
+     * @param options       The options for the certificate renewal
      * @return The Let's Encrypt purchase renewal response
      * @see <a href="https://developer.dnsimple.com/v2/certificates/#purchaseRenewalLetsencryptCertificate">https://developer.dnsimple.com/v2/certificates/#purchaseRenewalLetsencryptCertificate</a>
      */
-    public SimpleResponse<CertificateRenewal> purchaseLetsencryptCertificateRenewal(Number account, String domain, Number certificateId) {
-        return purchaseLetsencryptCertificateRenewal(account, domain, certificateId, false);
-    }
-
-    /**
-     * Purchase a Let's Encrypt certificate renewal.
-     * <p>
-     * This method creates a new renewal order. The order ID should be used to request the issuance of the certificate
-     * using `#issue_letsencrypt_certificate_renewal`.
-     *
-     * @param account       The account ID
-     * @param domain        The domain name or ID
-     * @param certificateId The certificate ID
-     * @param autoRenew     Enable auto-renewal of the certificate
-     * @return The Let's Encrypt purchase renewal response
-     * @see <a href="https://developer.dnsimple.com/v2/certificates/#purchaseRenewalLetsencryptCertificate">https://developer.dnsimple.com/v2/certificates/#purchaseRenewalLetsencryptCertificate</a>
-     */
-    public SimpleResponse<CertificateRenewal> purchaseLetsencryptCertificateRenewal(Number account, String domain, Number certificateId, boolean autoRenew) {
```

Comment:
I just wanted to note that this is going to introduce a breaking change to our client's API and I was wondering if you would consider maybe deprecating this one and adding a new one that takes the `CertificateRenewalPurchaseOptions` argument instead.

As for the `CertificateRenewalPurchaseOptions` value that we would use, I think it could default to `ECDSA`

Reply:
I think we are intentionally going to make this a breaking change, since adding support for `signature_algorithm` would not be just a bug fix patch anyway, so we'll also take the opportunity to remove some other deprecated parts. cc @DXTimer

Reply:
As Wilson shared we would like to take this opportunity to clear out some of the deprecated attributes for the certificates. The change for our users should not be difficult to implement.

Reply:
👍